apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -37,18 +37,73 @@
 }
 
 
-def filter_multiple_from_array(data, keys):
+class GradingMethodType(Enum):
```

Comment:
Nit: Move all type declarations above where the defaults are set (so TypedDicts and Enums), then group all defaults together.

Reply:
done